pkg/agent/agent.go

```diff
@@ -242,7 +242,7 @@ returns a spiffeid->registration entries map
 This map is used generated CSR for non-base SVIDs and update the agent cache entries
 */
 func (a *Agent) attest() (map[string]*common.RegistrationEntry, error) {
-	a.config.Log.Info("Preparing to attest against %s", a.config.ServerAddress.String())
+	a.config.Log.Info("Preparing to attest against", a.config.ServerAddress.String())
```

Comment:
Do we need a space at the end like the one below?

Reply:
Good catch. Spaces are necessary. Fixed.